docs/content/preview/migrate/yb-voyager/perform-migration.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Perform migration
+linkTitle: Perform migration
+description: Run the steps to ensure a successful migration.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview:
+    identifier: perform-migration
+    parent: yb-voyager
+    weight: 104
+isTocNested: true
+showAsideToc: true
+---
+
+This page describes the steps to perform and verify a successful migration to YugabyteDB. Before proceeding with migration, ensure that you have completed the following steps:
+
+- [Install yb-voyager](../../yb-voyager/install-yb-voyager/#install-yb-voyager).
+- [Prepare the source database](../../yb-voyager/install-yb-voyager/#prepare-the-source-database).
+- [Prepare the target database](../../yb-voyager/install-yb-voyager/#prepare-the-target-database).
+
+## Export and analyze schema
+
+### Export schema
+
+`yb-voyager export schema` command extracts the schema from the source database, converts it into PostgreSQL format (if the source database is Oracle or MySQL), and dumps the SQL DDL files in the `EXPORT_DIR/schema/*` directories.
+
+An example invocation of the command is as follows:
+
+```sh
+yb-voyager export schema --export-dir ${EXPORT_DIR} \
+        --source-db-type ${SOURCE_DB_TYPE} \
+        --source-db-host ${SOURCE_DB_HOST} \
+        --source-db-user ${SOURCE_DB_USER} \
+        --source-db-password ${SOURCE_DB_PASSWORD} \
+        --source-db-name ${SOURCE_DB_NAME} \
+        --source-db-schema ${SOURCE_DB_SCHEMA}
+```
+
+{{< note title="Note" >}}
+`source-db-schema` is only applicable for Oracle. Use this field only when migrating from Oracle to YugabyteDB for the [export schema](#export-schema), [analyze schema](#analyze-schema) and [export data](#export-data) phases.
```

Comment:
@amit-yb , a user would still need to make changes to the command for ${SOURCE_DB_TYPE} and other fields even if the user copy/pastes the command. So at the moment , since the note is right below the command, is that not sufficient to convey the information?